samples/src/client-file-download/NamedFileDownloader.ts

```diff
@@ -13,14 +11,9 @@ import { ClientStorage } from "@itwin/object-storage-core";
  * making this class cloud agnostic. {@link ClientStorage} instances are resolved
  * using instance name (which is described in the configuration as instanceName property)
```

Comment:
How now you resolve correct instance, if @named annotation is removed?

Reply:
I was looking at the main branch, https://github.com/iTwin/object-storage/blob/ad99f6189eace04640396974aaae42265b053548/samples/src/client-file-download/with-named-instances/App.ts#L34